### PR TITLE
[Inflector][String] Fixed singularize `edges` > `edge`

### PR DESCRIPTION
**Pull request by ruudk**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | yes
| New feature?  | no 
| Deprecations? | no
| License       | MIT

I noticed that calling `$inflector->singularize('edges');` returned `['edg', 'edges']`. 

It should be `edge`, only. See https://en.wiktionary.org/wiki/edge

### CODE REVIEW

File: src/Symfony/Component/Inflector/Tests/InflectorTest.php [hunk 1]
```diff
@@ -78,12 +79,12 @@ public function singularizeProvider()
...
             ['formulas', 'formula'],
             ['fungi', 'fungus'],
-            ['funguses', ['fungus', 'funguse', 'fungusis']],
+            ['fungi', ['fungus', 'funguse', 'fungusis']],
             ['garages', ['garag', 'garage']],
             ['geese', 'goose'],
             ['halves', ['half', 'halve', 'halff']],
             ['hats', 'hat'],
-            ['heroes', ['hero', 'heroe']],
+            ['heroes', ['hero', 'hero']],
```
Comment (stof) [inline src/Symfony/Component/Inflector/Tests/InflectorTest.php]: having it twice looks weird to me

Comment (ruudk) [reply src/Symfony/Component/Inflector/Tests/InflectorTest.php]: Oops, this happened on accident while backporting my fix from 5.2 to 4.4 branch. Updated.

Comment (ruudk) [reply src/Symfony/Component/Inflector/Tests/InflectorTest.php]: Somehow this happens by fabpot.io 🤔  

```
curl https://fabbot.io/patch/symfony/symfony/41105/8b2c0979505f1a43e6311111d48ebde8ba25de28/typos.diff | patch -p0
```

Comment (ruudk) [reply src/Symfony/Component/Inflector/Tests/InflectorTest.php]: Not sure how to solve this

Comment (stof) [reply src/Symfony/Component/Inflector/Tests/InflectorTest.php]: well, ignore it. It is totally expected to have typos in this test, as only one of the returned alternatives will be the valid word generally.

**Review by derrabus — APPROVED** [review]

**Review by stof — APPROVED** [review]

**Comment by derrabus** [comment]
Thank you @ruudk.

**Pull request merged**